SoftLayer/fixtures/SoftLayer_Network_Storage.py

```diff
@@ -1,4 +1,4 @@
-DUPLICATABLE_VOLUME = {
+SAAS_TEST_VOLUME = {
```

Comment:
In this case, SAAS was meant to represent "Storage As A Service". This test volume object is intended for use in tests relating to the Storage As A Service package. If it would help clear up possible misunderstanding, I am happy to add a comment to the source to indicate that SAAS corresponds to the Storage As A Service offering/package.